packages/@orchidui-vue/src/Form/SelectOptions/OcSelectOptions.stories.js

```diff
@@ -180,6 +180,7 @@ export const Default = {
       const errors = ref({});
 
       const onUpdateForm = (form, value = null) => {
+        console.log(form, value);
```

Comment:
Should this console.log be deleted?

Reply:
@mykhailo-diakovych console in the stories, no problem for now. in the future we make a preview model value/test component.